pom.xml

```diff
@@ -243,6 +243,14 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- Used for unit testing -->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.5.2</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Oh, I don't think we should be using junit 4 and 5 here.

Reply:
Updated.